uber/config.py

```diff
@@ -345,8 +348,14 @@ def _unrepr(d):
 c.make_dates(_config['dates'])
 
 c.PRICE_BUMPS = {}
+c.PRICE_LIMITS = {}
 for _opt, _val in c.BADGE_PRICES['attendee'].items():
-    c.PRICE_BUMPS[c.EVENT_TIMEZONE.localize(datetime.strptime(_opt, '%Y-%m-%d'))] = _val
+    try:
+        date = c.EVENT_TIMEZONE.localize(datetime.strptime(_opt, '%Y-%m-%d'))
+    except ValueError:
+        c.PRICE_LIMITS[int(_opt)] = _val
+    else:
+        c.PRICE_BUMPS[date] = _val
```

Comment:
Interesting.  So is the idea that the config file will now look something like

```
[badge_prices]
[[attendee]]

# Raise to $55 on Sept 1 or after the first 1000 badges are sold
2016-09-01 = 55
1000 = 55

# Raise to $60 on Oct 1 or after the first 2000 badges are sold
2016-09-01 = 60
2000 = 60

# etc
```

That seems sensible.  We should probably add some extra comments to ``configspec.ini`` to explain this.